zebra-chain/src/block/hash.rs

```diff
@@ -35,11 +47,47 @@ impl fmt::Debug for Hash {
         let mut reversed_bytes = self.0;
         reversed_bytes.reverse();
```

Comment:
Minor: unused variables
```suggestion
```

Reply:
thanks, it seems i expected the compiler to warn me about this. fixed in https://github.com/ZcashFoundation/zebra/pull/3754/commits/d35a302963f7e1ea67c33aca97ad137188e5ffc6